generator/carbonMonoxideSensor/main.go

```diff
@@ -0,0 +1,94 @@
+package main
+
+import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"math/rand"
+	"net/http"
+	"time"
+)
+
+type Event struct {
+	Signal  int
+	Name    string
+	Created time.Time
+	Event   map[string]string
+}
+
+type Status struct {
+	Status int
+}
+
+func main() {
+	createCarbonEvent()
+	http.ListenAndServe(":3003", nil)
+}
+
+func createCarbonEvent() {
+	s1 := rand.NewSource(time.Now().UnixNano())
+	r1 := rand.New(s1)
+	min := 35
+	max := 50
+	count := 0
+
+	for {
+		signal := r1.Intn(max-min) + min
+		creationTime := time.Now()
+		count = count + 1
+		if count == 36 {
```

Comment:
I discussed this with @polluxx already, please rewrite this to use `NormFloat64` like your colleagues did